CMakeLists.txt

```
@@ -24,6 +24,7 @@ cmake_policy(SET CMP0071 OLD)
 
 project(OpenInfraPlatform)
 
+message(STATUS "Please refer to .../Documentation/markdown/CMakeOptions.md for detail on all options that can be selected in the configuration.")

```

./ means "here" in path
```suggestion
message(STATUS "Please refer to ./Documentation/markdown/CMakeOptions.md for documentation on all options that can be selected in the configuration.")

```